Projects/Features/SearchFeature/Sources/Reactors/SearchReactor.swift

```diff
@@ -57,14 +59,27 @@ final class SearchReactor: Reactor {
             newState.typingState = state
         case let .updateText(text):
             newState.text = text
+        case let .fetchRecentText(text):
+            break
```

Comment:
이 친구는 나중에 구현하는건가요? 그러면 이런식으로 달아놓는게 좋을거같아요
```suggestion
        case let .fetchRecentText(text):
            #warning("추후 fetchRecentText 구현")
            break
```

Reply:
넵 반영하겠습니다.